libs/product/src/facades/product-grid/product-grid.facade.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { TestBed } from '@angular/core/testing';
+
+import { DaffProductGridFacade } from './product-grid.facade';
+import { MockStore } from '@ngrx/store/testing';
+import { State, reducers } from '../../reducers';
+import { Store, StoreModule, combineReducers } from '@ngrx/store';
+import { cold } from 'jasmine-marbles';
+import { DaffProductLoad } from '../../actions/product.actions';
+import { DaffProductGridLoad, DaffProductGridLoadSuccess } from '../../actions/product-grid.actions';
+
+describe('DaffProductGridFacade', () => {
+  let store: MockStore<Partial<State>>;
+  let facade: DaffProductGridFacade;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports:[
+        StoreModule.forRoot({
+          product: combineReducers(reducers),
+        })
+      ],
+      providers: [
+        DaffProductGridFacade,
+      ]
+    })
+
+    store = TestBed.get(Store);
+    facade = TestBed.get(DaffProductGridFacade);
+  });
+
+  it('should be created', () => {
+    expect(facade).toBeTruthy();
+  });
+
+  it('should be able to dispatch an action to the store', () => {
+    spyOn(store, 'dispatch');
+    facade.dispatch({ type: 'SOME_TYPE' });
+    expect(store.dispatch).toHaveBeenCalledTimes(1);
```

Comment:
And here (see other comment)

Reply:
addressed in 3185d1be860c060dac3f846dca340491ae2aff1a